src/Type/Definition/PhpEnumType.php

```diff
@@ -4,31 +4,35 @@
 
 use GraphQL\Error\SerializationError;
 use GraphQL\Utils\PhpDoc;
+use GraphQL\Language\AST\EnumValueDefinitionNode;
+use GraphQL\Language\AST\EnumTypeDefinitionNode;
+use GraphQL\Language\AST\EnumTypeExtensionNode;
 use GraphQL\Utils\Utils;
 
 /**
+ * @see EnumValueDefinitionNode
  * @phpstan-import-type PartialEnumValueConfig from EnumType
+ *
+ * @phpstan-type PhpEnumTypeConfig array{
+ *   name?: string|null,
+ *   description?: string|null,
+ *   enumPath: class-string<\UnitEnum>,
```

Comment:
```suggestion
 *   enumClass: class-string<\UnitEnum>,
```

path is something that relates to the filesystem